crates/relayer/src/chain/axon.rs

```diff
@@ -493,7 +550,7 @@ impl ChainEndpoint for AxonChain {
             .iter()
             .map(|seq| (*seq).into())
             .collect();
-        let height: Height = todo!();
+        let height = Height::new(u64::MAX, u64::MAX).unwrap();
```

Comment:
between axon and ckb, we don’t care about it, event between axon and cosmos as well, because the axon’s light client running on cosmos is also written by us, that’s my opinion